pkg/resolver/search.go

```diff
@@ -136,6 +232,9 @@ func (s *SearchResult) buildSearchQuery(ctx context.Context, count bool, uid boo
 	//WHERE CLAUSE
 	if s.input != nil && (len(s.input.Filters) > 0 || (s.input.Keywords != nil && len(s.input.Keywords) > 0)) {
 		whereDs = WhereClauseFilter(s.input)
+		if s.userAccess != nil {
+			whereDs = append(whereDs, buildRbacWhereClause(ctx, s.userAccess)) // add rbac
+		}
```

Comment:
```go
} else {
    panic("RBAC clause is required!")
}
```
Well `panic` might be a little extreme, but it could help during development.  😄 
All queries must include the RBAC filter or return an error.